models/queue.go

```diff
@@ -0,0 +1,22 @@
+package models
+
+import "time"
+
+type QueueType string
+
+const (
+	QueueType_Validate QueueType = "validate"
+	QueueType_Ready    QueueType = "ready"
+	QueueType_Batch    QueueType = "batch"
+	QueueType_Failure  QueueType = "failure"
+	QueueType_DLQ      QueueType = "dlq"
+)
+
+const QueueMaxLinger = 250 * time.Millisecond
+const QueueDefaultVisibilityTimeout = 5 * time.Minute
+const QueueMaxReceiveCount = 3
+
+type QueueRedrivePolicy struct {
```

Comment:
This is actually fine with me but just a note we are introducing aws specific language into the generic models. Thats okay with me but we may want to change this later (ex. using a different queue implementation)

Reply:
Yeah, good point, I can keep this under AWS stuff instead of making it part of the generic model, especially since it's not needed outside of initialization in `main`.

Reply:
Let me know what you think of the changes I just pushed.

Reply:
Though maybe we can do some more refactoring if/when we have more queue implementations.

Reply:
Ok, moved queue under AWS and did some minor renaming.

Reply:
looks good to me. we can always change things later and it would be easy to